### ISSUE TITLE
timetags of singles

### ISSUE THREAD

**Issue by Thomas-Jaeken** [Peter-Barrow/tangy#8]
At the moment the only way to collect time tags from a single channel is to calculate the coincidences of that channel with itself first.

Ideally, one could call buffer.timetags(channel=) directly.

**Comment by Peter-Barrow** [comment]
For reference, if you want to get the timetags for a specific channel the simplest way to do this would be the following. Where, for example, we want to find all occurances of channel 2 in the buffer:
```python
start = buffer.begin
stop = buffer.end
(channels, timetags) = buffer[start:stop]

channel_target = 2
timetags_channel_2 = timetags[channels == channel_target]
```

This can be wrapped up into a function though and added into the library

**Comment by Peter-Barrow** [comment]
[See](https://github.com/Peter-Barrow/tangy/blob/591958a479d0b2a2527a157583412421178dfbe9/tangy_src/_tangy.py#L875), new function added usage is as follows:

```
    def timetags(self,
                 channels: Optional[Union[int, List[int]]] = None,
                 start: Optional[int] = None,
                 stop: Optional[int] = None) -> List[Tuple[int, NDArray]]:
        """ Get the timetags for selected channels
        Args:
            channels (Optional[Union[int, List[int]]]): Channels to get
                timetags for, can be either a single channel (int) or many
                (list of ints). If no channels are set and value left as None,
                all channels will gathered.
            start (Optional[int]): Position in buffer to start collecting from,
                if None the first valid position in the buffer will be used.
            stop (Optional[int]): Position in the buffer to collect timetags
                up to, if left as None the end of the buffer will be used.

        Returns:
            (List[Tuple[int, NDArray]]): a list of tuples, each list element
                contains the channel index and ndarray of timetags for that
                channel

        Examples:
            Get all timetags for channels 0 and 1
            >>> timetags = buffer.timetags(channels=[0, 1])

            Get the first 10 timetags on channel 2
            >>> start = buffer.begin
            >>> stop = start + 10
            >>> channel_target = 2
            >>> timetags_channel_2 = timetags(channels=channel_target,
                                              start=start,
                                              stop=stop)

            Get the last 100 timetags on channel 4
            >>> stop = buffer.end
            >>> stop = stop - 100
            >>> timetags_channel_2 = timetags(channels=4,
                                              start=start,
                                              stop=stop)
        """
```

available in v0.9.1, update through pip
